internal/ad/internal_test.go

```diff
@@ -574,7 +568,6 @@ func TestParseGPOConcurrent(t *testing.T) {
 const SmbPort = 1445
 
 func TestMain(m *testing.M) {
-	defer testutils.StartLocalSystemBus()()
```

Comment:
are you sure about that one? What happens if we try to connect to a bus in ad and there is none, (like on the builder?)